tests/suites/test_suite_psa_crypto.function

```diff
@@ -1959,14 +1957,8 @@ void copy_key( int source_usage_arg, int source_alg_arg,
         psa_set_key_algorithm( &target_attributes, target_alg_arg );
 
     /* Copy the key. */
-    TEST_EQUAL( psa_copy_key( source_handle,
-                              &target_attributes, &target_handle ),
-                expected_status_arg );
-    if( expected_status_arg != PSA_SUCCESS )
-    {
-        TEST_EQUAL( target_handle, 0 );
-        goto exit;
-    }
+    PSA_ASSERT( psa_copy_key( source_handle,
+                              &target_attributes, &target_handle ) );
```

Comment:
For real implementation, would recommend asserting target_handle is non-zero upon return